.github/workflows/fast-forward-scheduled.yml

```diff
@@ -0,0 +1,24 @@
+name: Scheduled fast-forward from master to managed-services
+on:
+  schedule:
+    - cron: '0 7 * * 4'
+
+jobs:
+  check:
+    env:
+      PERIOD_DAYS: 21
+      STARTING_AT: "2022-10-27 00:00:00"
+    runs-on: ubuntu-latest
+    name: Decide whether it should call the job today
+    outputs:
+      RUN_TODAY: ${{ steps.check.outputs.RUN_TODAY }}
+    steps:
+      - id: check
+        run: |
+          echo "RUN_TODAY=$(($(($(($(date -d 00:00:00 +%s) - $(date -d '${{ env.STARTING_AT }}' +%s))) % $((60 * 60 * 24 * ${{ env.PERIOD_DAYS }} )))) == 0 ))" >> $GITHUB_OUTPUT
+  call:
+    name: Call the job
+    needs: check
+    if: ${{ needs.check.outputs.RUN_TODAY == 1 }}
```

Comment:
There's no way to configure cron to something like "every 3 Thursdays", so it's configured to run a job every Thrusday that  runs a script to check whether we are in the third Thursday, if true, then it calls the second job which makes the fast forward.

`RUN_TODAY` is `1` when we are in a week multiple of 3 starting by Thursday October 27. `0` Otherwise

Reply:
I forgot we decided on every 3rd week :(
Too bad we need 3 files now because of this but your solution is creative.